src/components/Search/Search.js

```diff
@@ -72,6 +72,9 @@ const Search = React.createClass({
     }
     return null;
   },
+  getDropdownVisible: function () {
```

Comment:
@rlr i think it's because you can't actually bind a getter to a different context so the context is still the original object rather than the class that gets returned